packages/clerk-js/src/ui/router/index.tsx

```diff
@@ -6,3 +6,6 @@ export * from './Route';
 export * from './Switch';
 
 export type { ParsedQs } from 'qs';
+export type ParsedQueryString = {
+  [key: string]: undefined | string | string[] | ParsedQueryString | ParsedQueryString[];
```

Comment:
❓ Are there any cases where the parsed queries would be anything different than `Record<string,string>`?
Can we confirm the above and simplify the type accordingly?